CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [1.1.6] - 2023-04-28
+### Changed
+
+* Refactored try blocks to try-with-resources for `DatabaseMetaDataUsingInfoSchema` class ([PR #388](https://github.com/awslabs/aws-mysql-jdbc/pull/388)).
+* Refactored resource shutdown for `CallableStatement` class ([PR #390](https://github.com/awslabs/aws-mysql-jdbc/pull/390)).
```

Comment:
Both are similar changes, can we put them together?

Refactored resource shutdown for `DatabaseMeta...` and `Callable...`